### PR TITLE
remove references to MySQL and Percona 5.6

### PR DESCRIPTION
**Pull request by deepthi**
## Description
MySQL 5.6 has been EOL for some time now.
Docker builds no longer work, so they have been disabled.
This PR removes the docker files for MySQL 5.6 and Persona 5.6, and references to them in various bootstrap / build files.

## Checklist

-   [x] "Backport me!" label has been added if this change should be backported
-   [x] Tests were added or are not required
-   [x] Documentation was added or is not required

### CODE REVIEW

**Comment by deepthi** [comment]
I have NOT removed the mysql56.cnf file. Removing that should be followed by `make embed_config`, which needs to wait until we are able to merge #10094

**Review by mattlord — APPROVED** [review]
Thank you! ❤️

**Pull request merged**
